install.sh

```diff
@@ -15,3 +15,6 @@ apt-get install -y vim
 add-apt-repository ppa:git-core/ppa
 apt-get update
 apt-get install -y git
+
+# comment out to mount via NFS outside VM:
+# apt-get install -y nfs-kernel-server
```

Comment:
No need to comment this out. The package can be installed even if the user doesn't use NFS.